### DISCUSSION TITLE
Switching workspace when game is in fullscreen causes me problems

### DISCUSSION THREAD

**Discussion by Kshitij-Tardalkar** [i3/i3#4641] [Q&A]
I am currently using i3wm on Arch Linux (zen kernel). I like to play games a lot and have been playing Dota 2 on Linux for the past few months. Recently I tried to join a friend's Discord while the game was running and thus changed my workspace. As soon as I changed the workspace I had an error where I got logged out of i3 and was shown to my LXDM screen. After logging in again, I tried running Dota2 again and this time it said "Only one instance can run at a time".  I tried doing the same thing and got the same result multiple times. I tried this with another game called 0AD and had the same issue again but this time another problem came up wherein the music of the game kept playing even though the game was closed. The only way to fix this was to restart my PC.

I don't think the problem is with full screen apps because when I run Zoom Meetings in full screen mode it never gives me such an error. Is there any way I can solve this issue

This is my OS info if you need it:
![image](https://user-images.githubusercontent.com/50481437/139649484-0d1a36b6-39ed-4127-80eb-d1ea4d665d20.png)

**Comment by Airblader** [comment]
> I had an error

What error, and by which application?

**Reply by Kshitij-Tardalkar** [reply]
By error I meant problem. I have explained the problem above. Sorry for the misunderstanding

**Comment by haddadnidal** [comment]
Hi, i am a dota 2 player myself. 
Dota is not fully optimized for linux users. had the same issue with gnome desktop. 
can you repeat the same process and when you relog in in I3 check if the dota2/steam client is still running since there is still music running

**Reply by Kshitij-Tardalkar** [reply]
TBH I never had this problem when I used Garuda Linux i3 when I switched workspaces. Started having the issue when I installed vanilla Arch and configured everything myself. Regarding the checking if it is running I don't really know how to do that, but on relaunching the game it said only one instance is allowed to run at a time, which may mean it was running in the background somewhere.

**Reply by haddadnidal** [reply]
How to check if the app is running you can do `ps aux | grep dota` or `ps aux | grep steam` the steam client is called steamhelper i think. im not on my gaming pc right now cant check.
